code/game/jobs/job/job.dm

```diff
@@ -33,7 +33,7 @@
 
 	var/datum/outfit/outfit = null
 	var/list/alt_outfits = null           // A list of special outfits for the alt titles list("alttitle" = /datum/outfit)
-	var/list/blacklisted_species = null   // A blacklist of species that can't be this job
+	var/list/blacklisted_species = list(SPECIES_VAURCA_BULWARK)   // A blacklist of species that can't be this job
```

Comment:
I am not a fan of setting a default here and then nulling that default out all over the place.
The current style is to explicitly set the species restrictions on the jobs that should have them.